conf/bk_server.conf

```diff
@@ -226,6 +226,15 @@ httpServerClass=org.apache.bookkeeper.http.vertx.VertxHttpServer
 #
 # permittedStartupUsers=
 
+# Certificate role based authorization for Bookie
+# To enable this option, bookieAuthProviderFactoryClass should be set to org.apache.bookkeeper.tls.BookieAuthZFactory
+# comma separated values of roles from the certificates OU field which we want to authorize for access
+#
+# Example settings:
+#   authorizedRoles=apache-pulsar-cluster-12,apache-kafka-cluster-2
```

Comment:
looks like 'kafka' is not useful in BK context :-)

Reply:
Fixed in [7276947](https://github.com/apache/bookkeeper/commit/72769477f52e4982081ae87a8e43da0a0e34e2d1)